frontend/src/config/controlPanelFeatures.ts

```diff
@@ -0,0 +1,22 @@
+import type { ControlPanelFeature } from '~/types';
+
+const CONTROL_PANEL_FEATURES: Record<ControlPanelFeature, boolean> = {
+  profile: false,
+  changePassword: false,
+  events: true,
+  images: true,
+  openingHours: true,
+  closedHours: true,
+  users: false,
+  roles: false,
+  gangs: false,
+  information: false,
+  documents: false,
+  recruitment: false,
+  sulten: false,
+  faq: false,
+};
+
+export function isControlPanelFeatureEnabled(f?: ControlPanelFeature) {
+  return !!f && CONTROL_PANEL_FEATURES[f] === true;
```

Comment:
Hadde vært nice om `f` ikke var nullable her... 👀 

```suggestion
export function isControlPanelFeatureEnabled(f?: ControlPanelFeature) {
  return !!f && CONTROL_PANEL_FEATURES[f];
```

Reply:
Ser ut til å fortsatt være nullable? Det hadde bare vært litt cleanere med en slik definisjon:

```ts
export function isControlPanelFeatureEnabled(f: ControlPanelFeature) {
  return CONTROL_PANEL_FEATURES[f];
}
```

Men det er ikke kritisk, så hvis det blir for mye styr kan du bare resolve denne tråden

Reply:
Done, ser ut som resten fortsatt fungerer selv om jeg bare endret funksjonen slik du foreslo